ui/fetch_ui_assets.sh

```diff
@@ -12,6 +12,6 @@
 # respective releases in "influxdata/ui" (OSS-2.0, OSS-2.1, etc). Those releases
 # are updated only when a bug fix needs included for the UI of that OSS release.
 
-curl -L https://github.com/influxdata/ui/releases/download/OSS-2.0/build.tar.gz --output build.tar.gz
+curl -L https://github.com/influxdata/ui/releases/download/OSS-2.0-Assets/build.tar.gz --output build.tar.gz
```

Comment:
If it isn't too late to suggest a different approach...

Why not mirror the strategy we use for the OSS releases, and tag this as the full `v<major>.<minor>.<patch>`? So we'd have a branch `OSS-2.0` and tags `OSS-v2.0.5`, `OSS-v2.0.6`... etc

Reply:
Phrased another way: `OSS-2.0-Assets` looks to me like a "floating" tag that will move around within the UI repo. That's convenient, but it prevents our builds from being reproducible.

Reply:
Definitely not too late for a different approach! And you're right about the `OSS-2.0-Assets` floating - it would need to get moved forward whenever something is backported into the `ui/OSS-2.0` branch, and that would make it difficult to know exactly what the UI was at the point of a certain OSS release.

With the `v<major>.<minor>.<patch>` strategy we will just need to update the download script a little more often - every patch release rather than every minor release. But probably worth it for being able to reproduce builds of OSS patch releases. I'll go ahead and make that update.